iree/compiler/Dialect/VMLA/Transforms/PreConversionLowering.cpp

```diff
@@ -277,6 +277,96 @@ class LowerBroadcastOp : public OpRewritePattern<mhlo::BroadcastOp> {
   }
 };
 
+// Lower mhlo::SortOp to an pseudo SortOp in the VMLA dialect. This
+// pseudo op generates a set of ordered indices for that array along the last
+// dimension. Then using a torch_index_select the values can be reordered to
+// support arbitrary inputs.
+//
+// TODO(suderman): This lowering only covers the case of ascending values, we
+// should support a separate descending value case by having separate
+// SortAscending and SortDescending operations.
+class LowerSortOp : public OpRewritePattern<mhlo::SortOp> {
+ public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(mhlo::SortOp op,
+                                PatternRewriter &rewriter) const override {
+    auto operand_ty = op.getOperand(0).getType().cast<RankedTensorType>();
+    bool last_dimension = (op.dimension() == -1) ||
```

Comment:
Use LLVM style naming throughout this function, to match the rest of `iree/compiler/` and this file
`last_dimension` -> `lastDimension`
`return_op` -> `returnOp`
etc.

Reply:
Constantly changing between repos hurts.....

Reply:
(fixed)